align_system/utils/kde_utils.py

```diff
@@ -18,14 +18,38 @@ def load_kde(target_kdma, norm='globalnorm'):
         target_kdma = target_kdma.to_dict()
 
     if norm == 'globalnorm':
-        target_kde_base64 = target_kdma['kdes']['globalnorm']['kde']
+        target_kde = kde_from_base64(target_kdma['kdes']['globalnorm']['kde'])
     elif norm == 'localnorm':
-        target_kde_base64 = target_kdma['kdes']['localnorm']['kde']
+        target_kde = kde_from_base64(target_kdma['kdes']['localnorm']['kde'])
     elif norm == 'rawscores':
-        target_kde_base64 = target_kdma['kdes']['rawscores']['kde']
+        target_kde = kde_from_base64(target_kdma['kdes']['rawscores']['kde'])
+    elif norm == 'priornorm':
+        norm_factor = 0.2 #TODO - make this a config paramter
+
+        # Load target KDE and get density
+        kde = kde_from_base64(target_kdma['kdes']['rawscores']['kde'])
+        linspace = np.linspace(0, 1, 1000)
+        density = _kde_to_pdf(kde, linspace)
+
+        # Get prior KDE and density
+        prior_data = [0.1]*100 + [0.3]*12 + [0.7]*8 + [0.9]*100 #TODO - get from train yamls rather than hard code (counts*4)
```

Comment:
Regarding getting the `prior_data` directly from the training data (rather than hardcoding); I believe the incontext database files could be used here (but I would want to pass in `prior_data` as a parameter here rather than a path to a training file or something to put less responsibility on this function).

That said, I don't think we need / want to worry about engineering this too much at this very moment since we already have long term refactoring planned for this part of the code, and we're coming up on the deadline soon.